custom-recipes/meaningcloud-deep-categorization/recipe.json

```
@@ -1,119 +1,157 @@
+            "name": "separator_input",
+            "label": "Input Parameters",
             "type": "SEPARATOR"
         },
         {
             "name": "column_name",
-            "label": "Column name",
+            "label": "Text column",
             "type": "COLUMN",
-            "description": "Column with the text to analyze.",
             "mandatory": true,
-            "columnRole": "input_dataset"
-        },        
-        {
-            "name": "predefined_model",
-            "label": "Use predefined model",
-            "type": "BOOLEAN",
-            "description": "Disable to classify with a user-defined model.",
-            "defaultValue": true
+            "columnRole": "input_dataset",
+            "allowedColumnTypes": [
+                "string"
+            ]
         },
         {
             "name": "language",
             "label": "Language",
-            "description": "Language of the text to analyze.",
+            "description": "List of supported languages for predefined models",
             "type": "SELECT",
-            "visibilityCondition" : "model.predefined_model == true",
             "selectChoices": [
-                {"value": "en", "label": "English"},
-                {"value": "es", "label": "Spanish"}
+                {
+                    "value": "en",
+                    "label": "English"
+                },
+                {
+                    "value": "es",
+                    "label": "Spanish"
+                }
             ],
             "defaultValue": "en"
         },
+        {
+            "name": "separator_configuration",
+            "label": "Configuration",
+            "type": "SEPARATOR"
+        },
         {
             "name": "model",
-            "label": "Model",
-            "description": "Model to use in the classification.",
+            "label": "Predefined model",
             "type": "SELECT",
-            "visibilityCondition" : "model.predefined_model == true",
             "selectChoices": [
-                {"value": "Emotion", "label": "Emotion Recognition"},
-                {"value": "IAB_2.0", "label": "IAB 2.0"},
-                {"value": "IntentionAnalysis", "label": "Intention Analysis"},                
-                {"value": "VoC-Banking", "label": "VoC - Banking"},
-                {"value": "VoC-Generic", "label": "VoC - Generic"},
-                {"value": "VoC-Insurance", "label": "VoC - Insurance"},
-                {"value": "VoC-Retail", "label": "VoC - Retail"},
-                {"value": "VoC-Telco", "label": "VoC - Telco"},
-                {"value": "VoE-ExitInterview", "label": "VoE - Exit Interview"},
-                {"value": "VoE-Organization", "label": "VoE - Organization"},
-                {"value": "VoE-Performance", "label": "VoE - Performance"}
-            ]
+                {
+                    "value": "IAB_2.0",
+                    "label": "Interactive Advertising Bureau 2.0"
```

```suggestion
                    "label": "IAB 2.0"
```
It's a well-known standard,  I think the abbreviation should be fine (plus visually it looks better in the drop down menu)

---

sure